mavis/bam/read.py

```diff
@@ -171,7 +171,7 @@ def get_samtools_version():
             mid = int(match.group('mid')) if match.group('mid') else 0
             minor = int(match.group('minor')) if match.group('minor') else 0
             return major, mid, minor
-    raise ValueError('unable to parse samtools version number')
+    raise ValueError('unable to parse samtools version number', proc)
```

Comment:
Should we make the string the same as the aligner for consistency?
 + raise ValueError("unable to parse blat version number from:'{}'".format(proc))

Reply:
but this is samtools and not the aligner

Reply:
sure I will add this in the other branch